frontend/src/utils/apiAxios.ts

```diff
@@ -29,6 +29,8 @@ export const apiAxios = () => {
     if (req.headers && keycloak.authenticated && keycloak.token) {
       req.headers.Authorization = `Bearer ${keycloak.token}`;
     }
+    console.log("req", req);
```

Comment:
is there a reason we are leaving this in??

Reply:
No, removing it now...